src/commands/sensor/commands/updatelocation.command.ts

```diff
@@ -3,10 +3,9 @@ import { ICommand } from '@nestjs/cqrs';
 export class UpdateSensorLocationCommand implements ICommand {
   constructor(
     public readonly sensorId: string,
-    public readonly x: number,
-    public readonly y: number,
-    public readonly z: number,
-    public readonly epsgCode: number,
```

Comment:
latitude + longitude zijn toch ook van een EPSG systeem? EPSG 4326?

Reply:
jawel, dat is gelijk het enige systeem wat officieel ondersteunt wordt in de geoJSON + mongoDB. Hoef je daarom niet op te slaan leek mij.